src/platform/Zephyr/BLEManagerImpl.cpp

```diff
@@ -245,7 +247,11 @@ void BLEManagerImpl::DriveBLEState()
     }
     else
     {
-        if (mFlags.Has(Flags::kAdvertising))
+        if (mFlags.Has(Flags::kAdvertising)
+#if CHIP_DEVICE_CONFIG_BLE_EXT_ADVERTISING
+            && mFlags.Has(Flags::kExtendedAdvertisingEnabled)
```

Comment:
Can you explain this change? Won't this cause that if the commissioning window is somehow closed earlier by a user application, the advertising won't be stopped?

Reply:
My bad, I misunderstood the part of this code